scripts/gen/windows-resources

```diff
@@ -6,6 +6,7 @@
 set -eu -o pipefail
 
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck source=/go/src/github.com/docker/cli/scripts/build/.variables
```

Comment:
The others can go but this particular one is needed because of `$SCRIPTDIR`

```
SC1090: Can't follow non-constant source. Use a directive to specify location.
```